scripts/build-local.py

```diff
@@ -164,6 +164,8 @@ def interrupt_handler(_signum, _frame):
                 distro = 'fedora'
             elif 'openSUSE' in url:
                 distro = 'opensuse'
+            else:
+                raise RuntimeError("URL ('{url}') was not handled?")
```

Comment:
This should be an f-string right?

`f"URL ('{url}') was not handled?"`

Reply:
Yes, thank you for catching this! Fixed on the force push.